### PR TITLE
Mark backup & restore operations as long running

### PR DESCRIPTION
**Pull request by vasanthrajams**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by heaths — APPROVED** [review]

**Review by docschmidt — APPROVED** [review]

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json [hunk 1]
```diff
@@ -33,6 +33,10 @@
           "FullBackup"
         ],
         "operationId": "FullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```
Comment (anuchandy) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json:39]: This is the first time I'm seeing the application of LRO annotation in a data-plane swagger. This annotation usually says that the applied operation follows the resource-manager LRO protocol. Here is the doc reference to this [annotation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation) and the mgmt SDKs have a special code path for LRO methods. Afaik the data-plane SDKs don't do anything automatically if it is produced from a swagger with this annotation. 

Looping in Johan @johanste from the API review board to comment on whether this annotation is recommended.

Comment (johanste) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json:39]: Data plane APIs also support LROs. We should verify that the generated code works (which relies on the fact that the service actually send an azure-async-operation header), but assuming that it does, the annotation should be there...

Comment (anuchandy) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json:39]: @johanste thanks! I checked .net &java vanilla code-gen, while those generate REST API method level for LRO, there is no specific logic for polling in azure-core hence Poller won't be "generated" as a return type. The data-plane SDK's for .net & java returns Poller from the convenience layer by wrapping the auto-gened REST API LRO method. This extension is considered only when SDK's are generated for ARM.


 @vasanthrajams could you add 200 response code also under responses, like:

```
"200": {
    "description": "OK"
},
```

this is to unblock the linter, though SDKs won't use them.

Comment (anuchandy) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json:39]: Sync-ed with Vasanta. The same pattern POST + 200 + azure-async-operation is already discussed with the team as part of another [PR](https://github.com/Azure/azure-rest-api-specs/pull/9647), the same linter error, in this case, is identified as a bug, hence we're good on this PR as well.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**